test/test_library.py

```diff
@@ -148,7 +148,7 @@ def test_album_fallback(self):
 class DestinationTest(_common.TestCase):
     def setUp(self):
         super().setUp()
-        self.lib = beets.library.Library(':memory:')
+        self.lib = beets.library.Library(':memory:', '~/.Music')
```

Comment:
I'm not quite sure I understand this—what does this directory argument do for the tests? Are we in any danger of writing files to this directory? Is there a reason it's located in the home directory (`~`) instead of in the root (`/`)?

Reply:
@sampsyo default directory is `~/Music` and the only reason why I had switched to `~/.Music` is to trigger a bug outside of MacPorts by adding second `.` in the path which is triggered a false fail in unit test.

Reply:
OK, got it, thanks. Do you think you could add a comment that clarifies why this is necessary? It would be great to leave clues for future maintainers—otherwise, this could look pretty mysterious if you don't have the context.

Reply:
@sampsyo here it is!